west.yml

```diff
@@ -0,0 +1,13 @@
+manifest:
+
+  remotes:
+    - name: mender
+      url-base: https://github.com/mendersoftware
+
+  projects:
+    - name: cjson-zephyr
+      submodules: true
+      remote: mender
+      path: modules/lib/cjson-zephyr
```

Comment:
Is this where it goes when fetching/checking out locally?

Reply:
Yes!

It is a bit arbitrary, but in `modules/lib` I found similar libraries like `zcbor`

Reply:
But actually, I made a mistake in the final submit: I wanted it to be `modules/lib/cjson` only, without the `-zephyr` suffix which in this context is superfluous. I'll amend this and merge :+1: